.gitignore

```diff
@@ -3,5 +3,6 @@
 *.iml
 *.ipr
 *.iws
+bulldozer.yml
```

Comment:
curious: why this?

Reply:
I can remove this. I was using this path for iterating locally with the configuration.

Reply:
all good. not sure if we should leave this, or encourage the use of the `config/*` folder for holding configs even for use while dev-ing. was just wondering cause it looked a bit unrelated

Reply:
policy-bot has an ignore for `config/policy-bot.yml`, which is the default config location for the server. That looks like it is also the default configuration location for Bulldozer (`config/bulldozer.yml`), so if we're going to add an ignore, let's make it match the server defaults.